app/templates/base.html

```diff
@@ -5,6 +5,7 @@
     {% block title %}
         <title>{{action}} {{data_type}}</title>
     {% endblock %}
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='_scss/view-styles.css') }}">
```

Comment:
Add your scss sheet to this file and it will get bundled
https://github.com/GSA/datagov-harvester/blob/harvest-email-on-errors/app/static/_scss/_uswds-theme-custom-styles.scss#L25

Reply:
Done.